scripts/new-package.js

```diff
@@ -0,0 +1,52 @@
+// a script that takes a string as value and copies the folder packages/template
+// to a new folder with the name of the value
+
+// Include process module
+const { exit } = require("process");
+const process = require("process");
+
+if (process.argv.length === 3) {
+  var projectName = process.argv[2];
+
+  // do some quick sanitization
+  var cleanProjectName = projectName.replace(/[|&;$%@"<>()+,]/g, "");
+
+  // copy the package/template folder to a new folder with the name of the value
+  // and then change the package.json file to the new name
+  const fse = require("fs-extra");
+  const fs = require("fs");
+  const path = require("path");
+  const templatePath = path.join(__dirname, "../packages/template");
+  const newProjectPath = path.join(
+    __dirname,
+    "../packages/" + cleanProjectName
+  );
+
+  // copy the template folder to the new project folder
+  fse.copySync(templatePath, newProjectPath, { dereference: true });
+
+  // change the package.json file to the new name
+  const packageJsonPath = path.join(newProjectPath, "package.json");
+  const packageJson = require(packageJsonPath);
+  packageJson.name = "@rnx-kit/" + cleanProjectName;
+  packageJson.description = "New package called " + cleanProjectName;
+  packageJson.homepage =
+    "https://github.com/microsoft/rnx-kit/tree/main/packages/" +
+    cleanProjectName +
+    "#readme";
+  packageJson.repository.directory = "packages/" + cleanProjectName;
+
+  fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2));
```

Comment:
We should also append an EOL to avoid the "missing newline at end of file" Git warning.
```suggestion
  fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2) + "\n");
```